### DISCUSSION TITLE
JET fires in FDS

### DISCUSSION THREAD

**Discussion by Muthu-V** [firemodels/fds#13086] [General]
Dear FDS developers ,  For sonic jet release fire consequence analysis there are several models available to get the Boundary conditions for an  expanded jet  such as jet diameter, jet velocity  and turbulence quantities k and epsilon at the outlet , and it can be used in CFD simulations with incompressible codes by ignoring the compressible part at the shock zones near the jet.  In FDS , I saw some interesting validation test case for LNG jet fires modelled with liquid droplets at high speeds. The code is stable and ran without any problems. Does this jet model has any limitation in using it for other fuels such as Hydrogen jet fires  or any other fuels ? The jet momentum and buoyant plume lifts can be reasonably predicted by this model  ?  Would appreciate your insights on this regard. Thanks

**Comment by rmcdermo** [comment]
With the caveats you already mentioned, that FDS cannot model detonations or predict mass flow through compressible nozzles, yes, the sonic regions are usually of very small volume and the heat release usually occurs at subsonic conditions and drives the buoyant flow.  There are also large-scale limits in terms of atmospheric effects (pyro-cumulus fires) that FDS does not handle.   But anything in the realm of the LNG cases (that is not mostly premixed) should be ok.

**Reply by Muthu-V** [reply]
Thank you Randy. Much appreciated your response.  To model hydrogen Jet fires  in FDS , where the leaks happen from High pressure segments ca. 600 bar , Can we specify  LH2 droplets at Temperature -253 C , droplet speeds ca. 1320 m/s  and retain the other jet parameters such as  jets angles etc.,  would be suffice ? Is there a way to exclude WSGGM radiation model and include only water vapour in the radiation calculation ? The main interest is to check the heat fluxes on the targets  in jet impingement zones and flame radiation in the nearby structures.

**Reply by rmcdermo** [reply]
I do not see any issues using WSGG or RADCAL.  The water vapor should be the dominant source.  If you use RADCAL you may need to tune the path length.  For radiation to distant targets, make sure you check a slice of 'INTEGRATED INTENSITY' at the target position (normal to incoming radiation) and use enough solid angles to make this smooth.  You can usually adjust the time increment and angle increment parameters to keep the cost of the calculation near the default (see user guide increment parameters).

Impinging jet heat fluxes are heavily influenced by convection, and there is a new convective heat transfer model specifically to handle this.  Search 'IMPINGING JET' in the user guide.  It makes a big difference at the jet impingement point.

One thing I would be cautious of is that particle injection usually has some offset from the source (if you are following a sprinkler example).  This offset can be important for the initial decay of momentum and you may need to tune your particle speed and direction to account for this.
